### DISCUSSION TITLE
Vote to make Naty a committer

### DISCUSSION THREAD

**Discussion by jcrist** [ibis-project/governance#6] [General]
I vote to make @ncclementi a committer per our [governance doc](https://github.com/ibis-project/governance/blob/main/governance.md). Naty has been working on the project for several months, and has been doing a great job maintaining the ibis project with an admirable level of care and attention to detail.

**Comment by cpcloud** [comment]
+1. Naty would be a great addition to the team!

**Comment by gforsyth** [comment]
+1 from me!

**Comment by jcrist** [comment]
With 3 votes, the motion carries. Welcome @ncclementi!

**Comment by ncclementi** [comment]
Thank y'all. I'm honored 💚

**Comment by cpcloud** [comment]
Done! Welcome to the team!
